package.json

```diff
@@ -43,7 +45,7 @@
     "commander": "^12.0.0",
     "convert-source-map": "^2.0.0",
     "dependency-graph": "^1.0.0",
-    "esbuild-wasm": "^0.20.0",
+    "esbuild": "^0.20.0",
```

Comment:
Should `esbuild-wasm` still be included for graceful degradation on unsupported archs?

Reply:
We removed this fallbacking in the CLI for esbuild builders